requirements.txt

```diff
@@ -8,4 +8,5 @@ py-evm==0.10.1b1
 pysha3>=1.0.0,<2.0.0; python_version<"3.9"
 requests==2.32.3
 safe-pysha3>=1.0.0; python_version>="3.9"
+types-requests==2.32.0.20240712
```

Comment:
I feel this should only be required for `dev` and not for production, am I right?

Reply:
Added in requirements-test ([here](https://github.com/safe-global/safe-eth-py/pull/1308/commits/1d71d495526362616735c2fdb81c507054333525)), to use it in [CI validation](https://github.com/safe-global/safe-eth-py/blob/68938a79ebc7ffdd0f4b33cc24ee19e05d436368/.github/workflows/python.yml#L67).